code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -675,12 +675,16 @@ GLOBAL_LIST_INIT(gas_id_to_canister, init_gas_id_to_canister())
 				logmsg = "Valve was <b>opened</b> by [key_name(usr)], starting a transfer into \the [holding || "air"].<br>"
 				if(!holding)
 					var/list/danger = list()
+					var/list/gaseslog = list()
 					for(var/id in air_contents.gases)
 						var/gas = air_contents.gases[id]
+						gaseslog[gas[GAS_META][META_GAS_NAME]] = gas[MOLES] //this may be the logging mechanism also i didnt capitalize right lol
```

Comment:
What's this comment talking about? nothing here looks wrong to me

Reply:
Oh I was commenting as back information to myself, I'll just update it to information on what the list is for